### ISSUE TITLE
Write up a better user-guide

### ISSUE THREAD

**Issue by eljobe** [eljobe/embed-a-cell#9]
This could simply be a wiki page in GitHub, or it could be a full-on website. Maybe at https://technodabble.com/embed-a-cell

**Comment by Androish** [comment]
Yes please, I have been trying to use this, but I am unable to get it to work, No clue how to select a spesific cell in a spreadsheet. Especially if it has several Sheets in it

**Comment by chr4004** [comment]
You need to specify the sheet ala Sheet1!A1:A1 to make it work.
